tests/fixtures/hypothesis_strategies.py

```diff
@@ -95,4 +109,21 @@ def custom_given(*args: Union[st.SearchStrategy, BaseModel], **kwargs):
     },
     ContextField: {"revision": False, "platform": False},
     ScoreResultField: {"raw": False, "min": False, "max": False},
+    UIPlayVideo: {
+        "event": custom_builds(PlayVideoEventField, currentTime=st.just(53.7))
+    },
+    UIPauseVideo: {
+        "event": custom_builds(PauseVideoEventField, currentTime=st.just(53.7))
+    },
+    UIStopVideo: {
+        "event": custom_builds(StopVideoEventField, currentTime=st.just(53.7))
+    },
+    UISeekVideo: {
+        "event": custom_builds(
+            SeekVideoEventField, old_time=st.just(21.4), new_time=st.just(36.9)
+        )
+    },
+    UISpeedChangeVideo: {
+        "event": custom_builds(SpeedChangeVideoEventField, new_speed=st.just("1.50"))
+    },
```

Comment:
Aren't the tests passing without these overwritten strategies? Why should they be overwritten?

Reply:
Yes it's a mistake